.env.example

```
@@ -26,6 +26,9 @@ EMAIL_SERVICE_EMAIL=""
 FIRST_RENEWAL_EMAIL_REMINDER_DAYS=28
 FIRST_RENEWAL_EMAIL_REMINDER_DAILY_RUN_TIME="1:05"
 
+# Expire registration scheduler configs
+EXPIRE_REGISTRATION_EXEMPTION_RUN_TIME="23:00"
```

We might as well go with what we'll be setting it to

```suggestion
EXPIRE_REGISTRATION_EXEMPTION_RUN_TIME="00:05"
```